configure.in

```diff
@@ -7778,7 +7793,7 @@ elif test "$GXX" = yes ; then
     dnl that they're deprecated but we still have to use them to support older
     dnl toolkit versions and leaving this warning enabled prevents seeing any
     dnl other ones
-    if test "$WXGTK3" = 1 -o "$wxUSE_MAC" = 1 ; then
+    if test "$WXGTK4" = 1 -o "$WXGTK3" = 1 -o "$wxUSE_MAC" = 1 ; then
```

Comment:
I don't think we want to disable deprecation warnings for GTK+4. At least not yet.